app/scripts/components/modeldetails.js

```diff
@@ -120,6 +120,14 @@ var exports = (function () {
             /* eslint-enable camelcase */
           }
         }
+      },
+      isIdle: {
+        deep: false,
+        handler: function () {
```

Comment:
I would rather use `handler: function (isIdle, oldIsIdle)` and then use `if (isIdle)` rather then this.isIdle, just to make sure that it did not change.